api/handler/acl.go

```diff
@@ -989,18 +993,38 @@ func addToList(operations []*astOperation, rec eacl.Record, target eacl.Target)
 	if found != nil {
 		if !groupTarget {
 			for _, key := range target.BinaryKeys() {
```

Comment:
Can be removed, I think. Only S3 gateways sets them and if it only sets accounts then we may not care about keys.

Reply:
Ok, removed